src/rcx/test/45_gcd.ok

```diff
@@ -6,18 +6,13 @@
 [INFO ODB-0133]     Created 350 nets and 978 connections.
 [INFO RCX-0431] Defined process_corner X with ext_model_index 0
 [INFO RCX-0029] Defined extraction corner X
-[INFO RCX-0008] extracting parasitics of gcd ...
 [INFO RCX-0435] Reading extraction model file 45_patterns.rules ...
 [INFO RCX-0436] RC segment generation gcd (max_merge_res 0.0) ...
 [INFO RCX-0040] Final 2656 rc segments
 [INFO RCX-0439] Coupling Cap extraction gcd ...
 [INFO RCX-0440] Coupling threshhold is 0.1000 fF, coupling capacitance less than 0.1000 fF will be grounded.
-[INFO RCX-0043] 1954 wires to be extracted
 [INFO RCX-0442] 48% completion -- 954 wires have been extracted
```

Comment:
I would be nice to combine these as

```[INFO RCX-0442] 48% completion -- 954 of 1954 wires have been extracted```

Reply:
Is a progress message approperiate here?

Isnt this always nearly instantaneous?